pkgs/top-level/all-packages.nix

```diff
@@ -21552,7 +21552,7 @@ with pkgs;
 
   ispc = callPackage ../development/compilers/ispc {
     xcode = darwin.xcode_14;
-    llvmPackages = llvmPackages_15;
+    llvmPackages = llvmPackages_17;
```

Comment:
Upstream goes as far as using their own patched version of llvm:
https://github.com/ispc/ispc/blob/main/llvm_patches/17_0_disable-A-B-A-B-and-BSWAP-in-InstCombine.patch
I'm not sure how to replicate that here

Reply:
Since "# This patch is needed for ISPC for Xe only", let's just hope to get away with the stock LLVM until the intersection of Xe and ISPC users start complaining.